### PR TITLE
implement batch entropy in python

### PR DESCRIPTION
**Pull request by jglidden**

### CODE REVIEW

File: loom/query.py [hunk 1]
```diff
@@ -161,9 +161,9 @@ def score(self, row):
             raise Exception('\n'.join(response.error))
         return response.score.score
 
-    def entropy(self, to_sample, conditioning_row=None, sample_count=None):
+    def entropy(self, to_samples, conditioning_row=None, sample_count=None):
         '''
-        Estimate the entropy
+        Estimate the entropy for to_sample in to_samples
```
Comment (fritzo) [inline loom/query.py]: "for each column in to_samples"?

Comment (fritzo) [inline loom/query.py]: Maybe this is a good time to rename `to_sample` to a noun like `requested_row` everywhere.

**Comment by fritzo** [comment]
+1 after comments

File: loom/query.py (hunk 1 above)
Comment (jglidden) [inline loom/query.py]: Yes, to_sample is no good. The word I want is "the argument to a probability function which is not the conditional part." I don't know a good term though.

**Pull request merged**